src/main/resources/templates/about.html

```diff
@@ -504,6 +505,31 @@ <h3 align="center">Roland YAP Hock Chuan</h3>
 						</div>
 					</div>
 				</div>
+				<div id="Intern" class="tab-pane fade">
+					<div class="container">
+						<div class="row team-members">
+							<div class="col-sm-3" style="margin:20px;width:250px;height:280px;padding:5px;border:2px dotted grey">
+								<!--<img src="images/team/18.png" class="img-circle" />-->
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)